mlflow/pyfunc/__init__.py

```diff
@@ -991,15 +1022,25 @@ def spark_udf(spark, model_uri, result_type="double", env_manager="local"):
             f"Illegal env_manager value '{env_manager}'.", error_code=INVALID_PARAMETER_VALUE
         )
 
+    if env_manager != "local" and os.name == "nt":
+        # TODO:
+        #  support windows system.
+        #   1. Address the issue killing mlflow server process when pyspark UDF task process died
+        #      (spark job canceled)
+        #   2. Addresss potential pyenv race condition issues on windows.
+        raise MlflowException(
+            f"`mlflow.pyfunc.spark_udf` env_manager value '{env_manager}' dose not support "
+            "windows system."
```

Comment:
I don't find proper approach for handling windows case (mainly the killing child process issue when parent process died by receiving SIGKILL). What about creating a FR ticket and ask microsoft guys to help ?

Reply:
Sounds great! :D. Can we update the exception content too and also add `error_code=INVALID_PARAMETER_VALUE`?

Reply:
Actually, how bad is it if we run anyway on Windows and just warn the user that there may be race conditions (are race conditions still a problem if we isolate the pip package cache?) and that the model server may continue running after executors terminate? I think this would be a better uX.

Reply:
I enable it under windows.
the related warning are printed here:
https://github.com/mlflow/mlflow/blob/e7caa0a1ffd47255f52b2818fad49d34ed332d99/mlflow/pyfunc/__init__.py#L1042

Reply:
> are race conditions still a problem if we isolate the pip package cache?

No.